### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by chrbinder** [Jeija/mesecons.net#20]
Hi! I really like the mesecons website for its design as well as the item types and documentation.
Do you think it would be possible to add other languages so that the mechanics discussed could be understood by non English speakers or by children?

**Comment by Jeija** [comment]
If someone wants to put the effort into translation the mesecons website with all text and item descriptions, I'm happy to accept pull requests. Unfortunately, I don't have the time to take of of that myself, sorry.

**Comment by chrbinder** [comment]
I may be able to provide German texts. Is there any possibility of a language switch or something like a selector already in place? I've got no clue on how to implement that.

**Comment by Jeija** [comment]
No, there is no such functionality yet, so someone else also needs to implement that.
